bin/install.sh

```diff
@@ -109,6 +109,12 @@ function install_cni() {
     fi
     bin/iop istio-cni istio-cni $IBASE/istio-cni/ ${ISTIO_CNI_ARGS}
     kubectl rollout status ds istio-cni-node -n istio-cni --timeout=$WAIT_TIMEOUT
+
+# Install kiali into namespace istio-kiali
+function install_kiali() {
```

Comment:
Has anyone ever executed this??? This is a function within a function. You introduce a `{` without  `}` which is never a good sign :)

I think we should enable some pre-commit testing or review more carefully. Our local pipeline is broken again.